compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2591,13 +2592,15 @@ setTranslationInfoAndRootConfig(func::FuncOp entryPointFn,
   // Ignore the tile sizes adjustment.
   auto pipeline = getTranslationInfo(entryPointFn).getPassPipeline().getValue();
   if (pipeline != DispatchLoweringPassPipeline::TransformDialectCodegen) {
-    if (failed(adjustTileSizesForUnPackOp(entryPointFn, rootOperation))) {
+    if (failed(adjustTileSizesForUnPackOp(entryPointFn, rootOperation)) &&
+        !hasMicrokernels(targetAttr)) {
```

Comment:
Can you split out all logic changes like this from the actual data-tiling-by-default flip, which should ideally be a 1-line PR?

Reply:
okay, I will do it!